src/App.tsx

```diff
@@ -1,173 +1,28 @@
-import { Loader } from './components/Loader';
```

Comment:
The Loader component import has been removed but it might be useful to show a loading indicator when fetching data from the API. Consider adding the Loader back and using it in the PeoplePage component. 😊